tools/osbuilder/image-builder/image_builder.sh

```diff
@@ -79,14 +79,15 @@ Usage: ${script_name} [options] <rootfs-dir>
 
 Options:
 	-h Show this help
-	-o path to generate image file ENV: IMAGE
-	-r Free space of the root partition in MB ENV: ROOT_FREE_SPACE
+	-o Path to generate image file. ENV: IMAGE
+	-r Free space of the root partition in MB. ENV: ROOT_FREE_SPACE
+	-f Filesystem type to use, only xfs and ext4 are supported. ENV: FS_TYPE
 
 Extra environment variables:
 	AGENT_BIN:      Use it to change the expected agent binary name
 	AGENT_INIT:     Use kata agent as init process
+	BLOCK_SIZE:     Use to specify the size of blocks in bytes. DEFAULT: 4096
 	IMAGE_REGISTRY: Hostname for the image registry used to pull down the rootfs build image.
-	FS_TYPE:        Filesystem type to use. Only xfs and ext4 are supported.
```

Comment:
Why `FS_TYPE` be deleted?

Reply:
Because filesystem type can be specified with the '-f' argument,  and also can specify it using 'FS_TYPE',  refer to https://github.com/kata-containers/kata-containers/blob/main/tools/osbuilder/image-builder/image_builder.sh#L477 .  If the parameters of the script can be obtained by getopts, perhaps it should be removed from the list of Extra env ariables,  such as env 'ROOT_FREE_SPACE'  and 'IMAGE'.